index.js

```diff
@@ -105,7 +166,24 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  let boxOffice = [];
+  if (!movies.length) {
+    return [];
+  }
+  genre = genre.toLowerCase();
+  for (let film of movies) {
+    let genre2 = film.genre.split(",");
+    for (let type of genre2) {
+      type = type.toLowerCase();
+      if (type.trim() == genre) {
+        //Forgot to account for the spaces after the comma - it works, but definitely would appreciate input on if there is a "better" way.
```

Comment:
One fix would be to change our `.split(",")` call to `.split(", ")` - this will change where in the string JS decides to split them to an array.  Also consider using a `regex`.   Take a look at the string `.replace()` method to get really fancy